packages/smooth_app/lib/pages/scan/ml_kit_scan_page.dart

```diff
@@ -62,8 +64,6 @@ class MLKitScannerPageState extends State<MLKitScannerPage> {
         ),
       );
     }
-
-    _startLiveFeed();
```

Comment:
This line might cause issues if everything is not ready

Reply:
Makes sense, but are you sure that it works with the current setup, it just calls this method from onResume but I'm unsure if onResume is called at first start of the app.

Reply:
This line is ok on both Android & iOS